composer.json

```diff
@@ -46,8 +47,7 @@
     },
     "scripts": {
         "test": [
-            "vendor/symfony-cmf/testing/bin/travis/phpcr_odm_doctrine_dbal.sh",
-            "SYMFONY_PHPUNIT_REMOVE=\"symfony/yaml\" vendor/bin/simple-phpunit"
+            "vendor/symfony-cmf/testing/bin/travis/phpcr_odm_doctrine_dbal.sh"
```

Comment:
the reason to use simple-phpunit was that for a while, phpunit shipped with a different version of symfony yaml and this led to use that version in the tests instead of the one installed with the project. the solution was to use simple-phpunit and tell it to strip the offending code from phpunit. when you take the branch from #208, is it enough to bump simple-phpunit to 6.3 or is there a problem with using simple-phpunit?